RELEASE_NOTES.md

```diff
@@ -2,8 +2,12 @@
 
 ## Version TBD
 _27 March 2017_
```

Comment:
Don't want future dates in here now; we're not sure when things will be pushed, and by having this here it'll make it harder to keep track